config/locales/en.yml

```diff
@@ -571,7 +571,7 @@ en:
           If you missed watching it live, we will have it up on the dashboard in a couple of days.
           Kindly help us improve our sessions with some quick feedback: https://svlabs.typeform.com/to/h7g9Om?faculty=%{faculty_name_escaped}&session=%{session_title}&date=%{session_date}
       send_session_reminders:
-        message: 'Reminder: "%{title}" will start in %{time_delta} (at %{time_exact}). Please check the Slack collective channel for the link to join session.'
+        message: "Reminder: \"%{title}\" will start in %{time_delta} (at %{time_exact}). Please check the Slack collective channel for the link to join session."
```

Comment:
This is something that is still handled differently from Prettier. I've filed [another issue](https://github.com/bodhish/pupilfirst-translator-vscode-ext/issues/2) in the extension repo.

Reply:
Have disabled prettier auto formating for en.yml file